source/components/molecules/GroupedList/GroupedList.tsx

```diff
@@ -66,11 +80,18 @@ const GroupedList: React.FC<Props> = ({
         property: LayoutAnimation.Properties.opacity,
       },
     });
+    // Sort whenever the summarylist is closed for editing
+    // i.e the 'Close' button is tapped
```

Comment:
Don't think this comment is needed; it doesn't seem to add any information not presented as code.

Reply:
Fixed